.travis.yml

```diff
@@ -0,0 +1,86 @@
+# Travis integration file.
+# Seattle supports Python 2.5 through 2.7, per
+#   seattle.poly.edu/wiki/ProgrammersPage
+#
```

Comment:
Addressed in https://github.com/SeattleTestbed/docs/blob/master/Contributing/BuildInstructions.md which supersedes the Trac wiki page.

However, we should note that we include Python 2.5 for testing still because it is what PlanetLab runs.